openstack/blockstorage/extensions/quotasets/requests.go

```diff
@@ -24,3 +24,61 @@ func GetUsage(client *gophercloud.ServiceClient, projectID string) (r GetUsageRe
 	_, r.Err = client.Get(u, &r.Body, nil)
 	return
 }
+
+// Updates the quotas for the given projectID and returns the new QuotaSet.
+func Update(client *gophercloud.ServiceClient, projectID string, opts UpdateOptsBuilder) (res UpdateResult) {
```

Comment:
Again, the variable names need to adhere to the style guide: s/res/r

Reply:
Thanks for catching that -- I missed it.

Self-note that this needs updated here, too: https://github.com/gophercloud/gophercloud/blob/master/openstack/compute/v2/extensions/quotasets/requests.go#L22